src/content/platform-integration/ios/setup.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Set up iOS development
+short-title: Set up iOS
+description: >-
+  Configure your development environment to
+  run, build, and deploy Flutter apps for iOS devices.
+---
+
+Learn how to set up your development environment
+to run, build, and deploy Flutter apps for iOS devices.
+
+:::note
+If you haven't set up Flutter already,
+visit and follow the [Get started with Flutter][] guide first.
+
+If you have already installed Flutter,
+ensure that it's [up to date][].
+:::
+
+[Get started with Flutter]: /get-started/install
+[up to date]: /install/upgrade
+
+## Install Xcode
+
+With Xcode, you can run Flutter apps on
+an iOS physical device or on the iOS Simulator.
+
+If you haven't done so already,
+install and set up [Xcode][].
+
+[Xcode]: https://developer.apple.com/xcode/
+
+Configure Xcode's command-line tools:
+
+```console
+$ sudo sh -c 'xcode-select -s /Applications/Xcode.app/Contents/Developer && xcodebuild -runFirstLaunch'
+```
+
+Sign the Xcode license agreement:
+
+```console
+$ sudo xcodebuild -license
+```
+
+## Install Rosetta, if applicable
+
+If you are developing on an [Apple Silicon][] (ARM) Mac,
+[install Rosetta 2][].
+
+```console
+sudo softwareupdate --install-rosetta --agree-to-license
+```
+
+[Apple Silicon]: https://support.apple.com/en-us/116943
+[install Rosetta 2]: https://support.apple.com/en-us/102527
+
+## Install CocoaPods
+
+If your app depends on any [Flutter plugin][]
+that uses native iOS code,
+install [CocoaPods][cocoapods].
+CocoaPods bundles various dependencies that
+support connecting Flutter to native iOS code.
+
+1. Install `cocoapods` following the
+   [CocoaPods installation guide][cocoapods].
```

Comment:
I like relying on their guide instead!

Since this is only one step now, consider dropping the list:

```suggestion
Install `cocoapods` following the
[CocoaPods installation guide][cocoapods].
```